web/test/pages/index-test.js

```diff
@@ -0,0 +1,10 @@
+import IndexPage from '../../src/pages/index'
```

Comment:
The module resolver isn't working in test files. :c

I think it might be because Next requires the webpack config to be done in the `next.config.js` file instead of a normal webpack config file?

Reply:
What do you want to be able to do here? `import IndexPage from 'pages/index';` ?

Reply:
`src/pages/index`

You can see here that it's working in the `src/` directory, but it wasn't working in `test/`
https://github.com/asleepysheepy/industry_tip_jars/blob/master/web/src/pages/_app.js#L4